### PR TITLE
[RfF] Improve makefile readability in NPQ

### PR DESCRIPTION
**Pull request by toolforger**
Request for Feedback: I'd like to hear whether I'm going in the right direction with this.

Running `make` and `make test` shows the same test failures as those on the master branch.
Not sure whether additional testing is advised; feedback appreciated.

Commits were built for individual reviewability.

Re-upload after fixes to embarrassing Makefile syntax mistakes.

### CODE REVIEW

File: tools/build/Makefile-common.in [hunk 1]
```diff
@@ -81,14 +81,22 @@ COMMON_NQP_SOURCES  = \
 
 NQP_COMBINED = NQP.nqp
 
+// No QAST_SOURCES, just one file, no need to combine
```
Comment (niner) [inline tools/build/Makefile-common.in]: I dind't know // worked as comment syntax in Makefiles. I've certainly never seen it used before. Are you sure that's compatible with alle relevant make implementations?

File: tools/build/Makefile-Moar.in [hunk 2]
```diff
@@ -38,19 +38,31 @@ P5QREGEX_MOAR     = NQPP5QRegex.moarvm
...
+  $(M_STAGE1_DIR)/$(MODULE_LOADER_MOAR) \
+  $(M_STAGE1_DIR)/$(CORE_SETTING_MOAR) \
+  $(M_STAGE1_DIR)/$(QASTNODE_MOAR) \
+  $(M_STAGE1_DIR)/$(QREGEX_MOAR) \
+  $(M_STAGE1_DIR)/$(HLL_MOAR) \
+  $(M_STAGE1_DIR)/$(MASTOPS_MOAR) \
+  $(M_STAGE1_DIR)/$(MASTNODES_MOAR) \
+  $(M_STAGE1_DIR)/$(QAST_MOAR) \
+  $(M_STAGE1_DIR)/$(P6QREGEX_MOAR) \
+  $(M_STAGE1_DIR)/$(NQP_MOAR)
```
Comment (niner) [inline tools/build/Makefile-Moar.in]: Maybe you should add a trailing backslash to the last line while you're at it, same as in Makefile-JVM.in?

Comment (toolforger) [inline tools/build/Makefile-Moar.in]: I wouldn't object to that, but it's not worth another round of testing and review IMO so I'll leave it as it is.
Put that in with the next PR you do ;-)

File: tools/build/Makefile-common.in (hunk 1 above)
Comment (toolforger) [inline tools/build/Makefile-common.in]: It does not. I would be surprised if it worked with any make implementation.
That's the embarrassing syntax mistake I talked about - it's fixed now ;-)

**Comment by toolforger** [comment]
Resolving the conflicts is more work than redoing the changes.

**Pull request closed without merging**